README.md

```diff
@@ -159,17 +159,10 @@ For use on an HPC cluster (rosalind, aspen) the workflow can be run as a job by
     module load nextflow
     ```
 
-2) Prepare file `cdc.confing` with the necessary `sge` submission parameters, add it to sub-directory [./conf/](./conf), and include it in [nextflow.config](./nextflow.config) with:  
```

Comment:
@jessicarowell couldn't this be either or? user could add their own cdc.conf with `includeConfig` OR just specify the scicomp reference cdc.conf.

Reply:
Yes, it could be either-or.  Just based on my conversations with scicomp, they'd rather people use the one they've preconfigured...and I think building one's own config file just adds unnecessary complexity.  But I can add back that line if you wish!